packages/bruno-electron/src/ipc/network/interpolate-vars.js

```diff
@@ -89,7 +89,7 @@ const interpolateVars = (request, envVariables = {}, runtimeVariables = {}, proc
         });
       } catch (err) {}
     }
-  } else if (contentType === 'multipart/form-data') {
+  } else if (contentType.indexOf('multipart/') === 0) {
```

Comment:
Wouldn't using [`.startsWith('multipart/')`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) make the code easier to read?

```suggestion
  } else if (contentType.startsWith('multipart/')) {
```

Reply:
Absolutelly.... I'm an old flashined JavaScript developer. Updated!